static/bidder-params/triplelift_native.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Triplelift Adapter Params",
+  "description": "A schema which validates params accepted by the Triplelift adapter",
+
+  "type": "object",
+  "properties": {
+    "inventoryCode": {
+      "type": "string",
+      "description": "TripleLift inventory code for this ad unit (provided to you by your partner manager)"
+    },
+    "floor" :  {"description" : "the bid floor", "type": "number" }
```

Comment:
Do we want to specify a currency In the description field of `"floor"`? It is not required but other adapters do that.

Reply:
just `"currency":"usd"` ,for example?

Reply:
No, just in the description string. If you think it is necessary, or desirable, something like:
```
  8     "inventoryCode": {
  9       "type": "string",
 10       "description": "TripleLift inventory code for this ad unit (provided to you by your partner manager)"
 11     },
 12  -  "floor" :  {"description" : "the bid floor", "type": "number" }
 13  +  "floor" :  {"description" : "the bid floor, USD <or any other currency> by default", "type": "number" }
 14   },
 15   "oneOf": [{
 16     "oneOf": [{
 17       "required": ["inventoryCode"]
 18     }] }],
 19   "not": {
 20     "required": ["floor"]
 21   }
 22 }
static/bidder-params/triplelift_native.json [+]
```

Reply:
Entirely optional

Reply:
done